### PR TITLE
use github.com/cadmuxe/kubernetes=release-1.11-calico-fix repo

### PR DESCRIPTION
**Pull request by cadmuxe**
Use repo that contains https://github.com/cadmuxe/kubernetes/commit/43e1b198cd5c2f2d4fe103f00b7a5eba755dd06d

### CODE REVIEW

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 1]
```diff
@@ -547,10 +547,12 @@ periodics:
...
     - args:
       - "--timeout=340"
-      - "--repo=k8s.io/kubernetes=release-1.11"
+      - "--repo=github.com/cadmuxe/kubernetes=release-1.11-calico-fix"
+      - --build=bazel
       - --scenario=kubernetes_e2e
       - --
-      - --extract=ci/latest
+      - --extract=local
+      - --stage=gs://kubernetes-release-pull/ci/ci-kubernetes-e2e-gce-netd-calico
```
Comment (cadmuxe) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: @krzyzacy I want to build and stage it to gs://kubernetes-release-pull/ci/ci-kubernetes-e2e-gce-netd-calico  and reuse it in the other testgrid. is it correct?

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: just do `--extract=gs://kubernetes-release-pull/ci/ci-kubernetes-e2e-gce-netd-calico`, you are not staging anything here

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 2]
```diff
@@ -597,10 +596,9 @@ periodics:
     containers:
     - args:
       - "--timeout=340"
-      - "--repo=k8s.io/kubernetes=release-1.11"
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: you need --bare, like [these jobs](https://github.com/kubernetes/test-infra/blob/master/config/jobs/gke/gke-staging.yaml#L9-L13) (presubmit should catch this ?!)

**Comment by krzyzacy** [comment]
/lgtm
:eyes:

**Pull request merged**